website/pages/api-docs/auth/ldap/index.mdx

```diff
@@ -47,6 +47,10 @@ This endpoint configures the LDAP auth method.
   verification - insecure, use with caution!
 - `certificate` `(string: "")` – CA certificate to use when verifying LDAP server
   certificate, must be x509 PEM encoded.
+- `client_tls_cert` `(string "")` - Client certificate to provide to the LDAP
+  server, must be x509 PEM encoded (optional).
+- `client_tls_key` `(string "")` - Client certificate key to provide to the LDAP
+  server, must be x509 PEM encoded (optional).
```

Comment:
```suggestion
  server. Must be x509 PEM encoded (optional).
```

Reply:
Well, I guess this `, must be` format is consistent with the rest of the docs, but it reads funny to me. Accept or decline as you will